packages/web-components/src/components/masthead/cloud/cloud-masthead-composite.ts

```diff
@@ -249,6 +249,8 @@ class DDSCloudMastheadComposite extends DDSMastheadComposite {
     }
     this._loadTranslation?.(language, dataEndpoint).catch(() => {}); // The error is logged in the Redux store
     this._loadUserStatus?.(this.authMethod);
+
+    this.style.zIndex = '999';
```

Comment:
this is a temp fix until we figure out why we can't set styles to the `cloud-masthead-container` with stylesheets